packages/components/header/_header-transactional.scss

```
@@ -3,12 +3,9 @@
 */
 
 .nhsuk-header__transactional-service-name {
+  margin-bottom: #{nhsuk-spacing(1) * -1};
```

Elsewhere the scss uses `-()` for negative numbers, which feels a bit more intuitive than multiplying by -1?

```suggestion
  margin-bottom: -(nhsuk-spacing(1));
```